README.md

```diff
@@ -1,2 +1,93 @@
 # go-health-check
 Go package to create health-check page
+
+### About
+
+This library provide simple health check for:
+
+- SQL DB
+- Memcached
+- Http
+- MongoDB
+- Composite Checkers
+- Check by `func`
+
+Use [fasthttp](https://github.com/valyala/fasthttp) for HTTP and [fasthttproute](https://github.com/buaazp/fasthttprouter) for routing
+
+### Example
+
+```go
+import (
+	"time"
+	"fmt"
+	"database/sql"
+	
+	"github.com/rainycape/memcache"
+	"github.com/ronte-ltd/go-health-check/checkers"
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	fmt.Println("Start.")
+	mc, _ := memcache.New("127.0.0.1:11211")
+	mcChecker := checkers.NewMemcachedChecker("memcache", mc)
+	defer mc.Close()
+
+	http := checkers.NewHttpChecker("Yandex", "https://ya.ru")
+
+	DB, _ := sql.Open("postgres", "postgres://postgres:postgres@127.0.0.1:5432/postgres?sslmode=disable")
+	defer DB.Close()
+	dbChecker := checkers.NewDBChecker("Postgres", DB)
+
+	funcChecker := checkers.NewFuncChecker("MyChecker", MyChecker)
+	composite := checkers.SimpleChecker("CompositeChecker")
+	composite.AddChecker(&mcChecker)
+	composite.AddChecker(&http)
+	composite.AddChecker(&funcChecker)
+	composite.AddChecker(&dbChecker)
+
+	handler := checkers.NewHandler("localhost:11911")
+	go handler.AddRoute("/health", &composite)
```

Comment:
the library must provide an object that will be used as a standard net/http Handler. Application that uses the library will add routing itself.

Reply:
I hid implementation routing inside `checker`. Now you can call `checker.Handle("some_add", "/health_route")` for running server and routing.